adafruit_featherwing/minitft_featherwing.py

```diff
@@ -74,11 +74,16 @@ def __init__(self, address=0x5E, i2c=None, spi=None):
         self._backlight.duty_cycle = 0
         self._ss.pin_mode_bulk(self._button_mask, self._ss.INPUT_PULLUP)
         displayio.release_displays()
-        display_bus = displayio.FourWire(spi, command=board.D6, chip_select=board.D5)
+        spi.try_lock()
```

Comment:
I don't think you can assume the try_lock() will be "True" immediately - you need tp test and wait for it to be True - possibly with a timeout and error.

Reply:
the timeout is probably overkill, but I recommend using 
```
while not spi.try_lock():
    pass
```
to make sure the lock state is True.

Reply:
I changed line to what you suggested.